PyPDF4/pdf.py

```diff
@@ -62,14 +62,6 @@
     ConvertFunctionsToVirtualList
 from .utils import isString, pypdfUnicode, pypdfOrd, pypdfStr,\
     formatWarning, pypdfBytes as b_
-
-if version_info < (2, 4):
-   from sets import ImmutableSet as frozenset
-
-if version_info < (2, 5):
-    from md5 import md5
-else:
```

Comment:
Deleting this block introduced a small bug that was later fixed within the same PR.